api/can_server/views.py

```diff
@@ -49,10 +50,41 @@ def get_dbc_files(request):
     dbc_files = DbcFile.objects.all()
     dbc_file_serializer = DbcFileSerializer(dbc_files, many=True)
     files_list = []
+
     for entry in dbc_file_serializer.data:
         db = cantools.database.load_string(entry['FileData'])
         files_list.append(entry['FileName'])
     return JsonResponse(
-                {'response': files_list},
-                status=200
-            )
+        {'response': files_list},
+        status=200
+    )
+
+
+# example PUT request: {"bustype": "virtual", "channel":"vcan", "bitrate":"800000"}
+@api_view(['PUT'])
+def change_settings(request):
```

Comment:
This looks good, though we should try to write some tests at some point for these API requests.